src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -360,7 +365,7 @@ def is_quadratic_twist(self, other):
             2
             sage: E1.is_quadratic_twist(E1)
             1
-            sage: type(E1.is_quadratic_twist(E1)) == type(E1.is_quadratic_twist(E2))   #trac 6574
+            sage: type(E1.is_quadratic_twist(E1)) == type(E1.is_quadratic_twist(E2))   # Issue #6574
             True
```

Comment:
OK, but I plan someday to standardize `Issue #6574`, `github issue #6574`, `trac #6574`, `pr #6574` etc to `issue #6574`.